### PR TITLE
Fix push to undefined

### PR DESCRIPTION
**Pull request by tarcisiozf**
### Description:
...

### Breaking changes:
- [ ]

### New features:
- [ ]

### Fixes:
- [X] https://app.asana.com/0/1200227858408358/1201037680350668/f

### PR status:
- [ ] Version bumped
- [ ] Change-log updated
- [ ] Tests added or updated
- [ ] Documentation updated

### CODE REVIEW

**Review by avsek477 — APPROVED** [review]
lgtm

**Review by ghost — APPROVED** [review]
lgtm

File: lib/manager.js [hunk 1]
```diff
@@ -402,7 +402,6 @@ class Manager extends EventEmitter {
       : []
 
     if (!_isFunction(func)) {
-      debug('plugin %s is missing a handler for type %s', id, type)
```
Comment (vigan-abd) [inline lib/manager.js:405]: @tarcisiozf why we're removing this debug?

Comment (tarcisiozf) [reply lib/manager.js:405]: this was a misconception when I first added this debug log, a plugin does not need to listen to all events in the manager lifecycle, this debug message floods the stdout with false positive errors.

**Pull request closed without merging**